README.md

```diff
@@ -113,6 +114,7 @@ docker stop altinn-runtime
 ```
 
 Navigate to the Runtime folder. Build and run the code.
+**Important:** You must first have run the Designer builds and runs in order to get successfull runs on the following commands for Runtime. 
```

Comment:
I do not understand "runs". Perhaps you can use comands?

Reply:
Good tip. Now fixed a new and better text explaining this.